conans/util/windows.py

```diff
@@ -1,3 +1,4 @@
+import nt
```

Comment:
I think this is mostly a private Python implementation detail, I am concerned about using this directly, as this is not public documented Python library: https://docs.python.org/3/library/

Reply:
I've found the recommendation here: https://stackoverflow.com/questions/19023238/why-python-uppercases-all-environment-variables-in-windows
`nt` module is also mentioned here: https://www.oreilly.com/library/view/python-standard-library/0596000960/ch12s12.html